pytest.ini

```diff
@@ -12,3 +12,9 @@ testpaths =
     tests/where_test.py
 norecursedirs = .git dist build *egg* tests/deprecated/*
 python_functions = test*
+env =
+    ARKOUDA_SERVER_HOST=localhost
+    ARKOUDA_SERVER_PORT=5555
+    ARKOUDA_FULL_STACK_TEST=True
+    ARKOUDA_NUMLOCALES=2
```

Comment:
In our nightly XC/CS testing we `export ARKOUDA_NUMLOCALES=16` -- does the env var take priority, or does the value in the .ini file? (Based on a quick test it seems like the value in the ini file)

Reply:
Deferring to you for pytest expertise, but maybe we can prefix these with `D:` to not override command line settings?

Reply:
@ronawho You are correct and it sounds like it's dorking up your nightly test so I will take that out of pytest.ini.

Reply:
I'm find leaving it in if the `.ini` style is what python programmers will be used to. I think it'd just need to be `D:ARKOUDA_NUMLOCALES=2` so that command line settings won't be overwritten

Reply:
@ronawho gotcha, okay